### PR TITLE
[#88] Add relabel for instance

### PR DESCRIPTION
**Pull request by 532910**
Closes https://github.com/nspcc-dev/nspcc-infra/issues/88

Signed-off-by: Sergio Nemirowski <sergio@nspcc.ru>

### CODE REVIEW

File: config/testnet/docker-compose.yml [hunk 1]
```diff
@@ -5,26 +5,23 @@ services:
...
-      - NEOFS_GRPC_ENDPOINT=<BIND_ADDRESS>
-      - <NEOFS_NODE_ATTRIBUTE_1>=UN-LOCODE:<XX YYY>
-      - NEOFS_NODE_ATTRIBUTE_2=Price:100000
 
   vmagent:
     image: victoriametrics/vmagent:v1.61.0
     container_name: vmagent
     depends_on:
       - storage01
+    env_file: env
```
Comment (realloc) [inline config/testnet/docker-compose.yml:21]: Whre is `env` file? Shouldn't that be `node_config.env`?

**Comment by realloc** [comment]
The README.md is not changed according to the new compose file struture. Please fix.

**Review by realloc — CHANGES_REQUESTED** [review]
The README.md is not changed according to the new compose file structure. Please fix.

File: config/testnet/docker-compose.yml (hunk 1 above)
Comment (532910) [reply config/testnet/docker-compose.yml]: Yep, my bad, fixed.

**Comment by 532910** [comment]
> The README.md is not changed according to the new compose file structure. Please fix.

Fixed.

File: config/testnet/docker-compose.yml [hunk 2]
```diff
@@ -5,26 +5,23 @@ services:
   storage01:
     image: nspccdev/neofs-storage-testnet:0.21.1
     container_name: neofs-testnet
+    env_file: node_config.env
     network_mode: host
     restart: always
     volumes:
-      - neofs_storage:/storage
+      - <NEOFS_STORAGE>:/storage
```
Comment (alexvanin) [inline config/testnet/docker-compose.yml:12]: In README.md we have

```
NeoFS objects will be stored on your machine. By default, docker-compose 
configured to store objects in named docker volume `neofs_storage`. You can 
specify directory on the filesystem to store objects there.
```

Can we keep object storage in a docker volume by default?

Comment (532910) [reply config/testnet/docker-compose.yml]: agree, fixed

**Review by alexvanin — APPROVED** [review]

**Pull request merged**